app/src/main/java/org/mifos/mobilebanking/ui/fragments/UserProfileFragment.java

```diff
@@ -153,16 +154,26 @@ public void showUserDetails(Client client) {
         tvGroups.setText(getGroups(client.getGroups()));
         tvClientClassification.setText(client.getClientClassification().getName());
         tvPhoneNumber.setText(client.getMobileNo());
-        tvDOB.setText(DateHelper.getDateAsString(client.getDobDate()));
+        if (client.getDobDate().size() != 3) {  // no data entry in database for the client
+            tvDOB.setText("No DOB found.");
```

Comment:
@ankurs287 we are not using any hardcoded string anywhere. So move this string to `strings.xml` and you string value must be `No Date of Birth`

Reply:
okay. I will do that.